client/src/App.js

```diff
@@ -18,12 +25,20 @@ export default class App extends Component {
     this.setState({ savedList });
   };
 
-  render() {
-    return (
-      <div>
-        <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
-      </div>
-    );
-  }
+    render() {
+	return (
+	    <div>
+              <SavedList list={this.state.savedList} />
+		<Route
+		  exact
+		  path='/'
+		  component={MovieList}
+		  />
+		<Route
+		  path='/movies/:id'
+		  component={Movie}
```

Comment:
This works, but it is better to use render than component. So:
render={props => <Movie {...props} />}
It's technical, but explanation here:
https://stackoverflow.com/questions/48150567/react-router-difference-between-component-and-render
TLDR - component creates new component each time, render does not. Render is a bit faster.